src/UserService.Mappers/Db/DbUserCredentialsMapper.cs

```diff
@@ -24,7 +24,8 @@ public class DbUserCredentialsMapper : IDbUserCredentialsMapper
                 Salt = salt,
                 UserId = request.UserId,
                 PasswordHash = passwordHash,
-                IsActive = true
+                IsActive = true,
+                CreatedAtUtc = DateTime.UtcNow
```

Comment:
did you forget about CreatedBy or in this models we haven't this property?

Reply:
in this models we haven't this property